src/tactic/hint.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import tactic.tidy
+
+namespace tactic
+
+namespace hint
+
+/-- An attribute marking a `tactic unit` or `tactic string` which should be used by the `hint` tactic. -/
+meta def hint_attribute : user_attribute := {
```

Comment:
```suggestion
@[user_attribute] meta def hint_attribute : user_attribute := {
```

Then you can delete the `attribute.register` line below.

Reply:
Done.